modules/lvgl/include/lvgl_display.h

```diff
@@ -49,6 +49,10 @@ int set_lvgl_rendering_cb(lv_disp_drv_t *disp_drv);
 
 void lvgl_flush_display(struct lvgl_display_flush *request);
 
+#if CONFIG_LV_Z_AREA_X_ALIGNMENT_WIDTH
```

Comment:
Can you check that `!= 1` instead or introduce a kconfig symbol `LV_Z_USE_ROUNDER_CB` with no prompt that is set if above is not 1?

Reply:
The introduction of `LV_Z_USE_ROUNDER_CB` Kconfig should make more essence. I added support for frame Y alignment, as well.